### PR TITLE
[ Fixes #187373269 #187355094 #187355101 ] fixing -user-login-and-registration-and-google-login-emails-theme

### PR DESCRIPTION
**Pull request by Celestin4**
**1. What does this PR do?**

Fixes fx-user-login-and-registration-and-emails-theme

**2. Description of task to be completed**

This is bug fix pr that raised for fixing themes in HTML template, user login by google to allow use of google access token. It also fixed verifying user so that we may handle redirection to appropriate client page.

**3. How should this be manually tested?**

Clone the repository.
Checkout to the branch.
Install necessary dependencies.
Start the development server.
Utilize Swagger documentation to interact with the API.

### CODE REVIEW

**Comment by UmuhireJessie** [comment]
Can you resolve conflicts and rename your commit message? Consult https://www.pivotaltracker.com/help/articles/github_integration/

File: src/controllers/userControllers.ts [hunk 1]
```diff
@@ -194,7 +194,8 @@ export default class UserController {
   }
 
   static async registerUserGoogle(req: any, res: any) {
-    const data = req.user._json;
+    const data = req.body;
+    console.log(data);
```
Comment (UmuhireJessie) [inline src/controllers/userControllers.ts]: remove this console

File: src/helps/nodemailer.ts [hunk 2]
```diff
@@ -44,7 +44,7 @@ async function sendEmail(
   try {
     await transporter.sendMail(message);
   } catch (error) {
-    console.error('Error sending email:', error);
+    // console.error('Error sending email:', error);
```
Comment (UmuhireJessie) [inline src/helps/nodemailer.ts]: On error, you may want to know what is happening

**Review by UmuhireJessie — CHANGES_REQUESTED** [review]
Check my comments and resolve conflicts

**Review by UmuhireJessie — APPROVED** [review]

**Review by Celse13 — APPROVED** [review]

**Review by pacifiquemboni — APPROVED** [review]

**Review by shema-surge — APPROVED** [review]

**Pull request merged**